src/network/networkd-dhcp6.c

```diff
@@ -181,20 +182,21 @@ static void log_dhcp6_address(Link *link, const Address *address) {
         if (existing->source == NETWORK_CONFIG_SOURCE_NDISC)
                 by_ndisc = true;
 
-        log_link_warning(link, "DHCPv6 address %s/%u (valid %s, preferred %s) conflicts the address %s/%u%s.",
+        log_link_warning(link, "DHCPv6 address %s/%u (valid %s, preferred %s) conflicts the address %s/%u%s. Ignoring the DHCPv6 address.",
```

Comment:
Pre-existing condition, but "a conflicts b" is very awkward. "a conflicts with b" would be better.
Maybe change this to:
```C
log_link_warning(link, 
                        "Ignoring DHCPv6 address %s/%u (valid %s, preferred %s) which conflicts with %s/%u%s.",
                         strna(buffer), address->prefixlen,
                         FORMAT_LIFETIME(address->lifetime_valid_usec),
                         FORMAT_LIFETIME(address->lifetime_preferred_usec),
                         strna(buffer), existing->prefixlen,
                         by_ndisc ? " assigned by NDisc" : "");
if (by_ndisc)
   log_link_info(link, "Hint: use IPv6Token= setting to change the address generated by NDISC or set UseAutonomousPrefix=no.");
```